sql/conseil.sql

```diff
@@ -918,7 +918,7 @@ ALTER TABLE ONLY ethereum.transactions
 
 -- Table is based on eth_getLogs from https://eth.wiki/json-rpc/API
 CREATE TABLE ethereum.logs (
-  address text NOT NULL PRIMARY KEY,
+  address text NOT NULL,
```

Comment:
Is this a shortcoming or does it have specific reason for not being unique?

Reply:
There can be multiple log entries corresponding to one address, this is my mistake.